### PR TITLE
Update ESR version

### PR DESCRIPTION
**Pull request by amyblais**

### CODE REVIEW

File: app/constants/calls.ts [hunk 1]
```diff
@@ -6,9 +6,9 @@ import {toMilliseconds} from '@utils/datetime';
 const RefreshConfigMillis = toMilliseconds({minutes: 20});
 
 const RequiredServer = {
-    FULL_VERSION: '6.3.0',
-    MAJOR_VERSION: 6,
-    MIN_VERSION: 3,
+    FULL_VERSION: '7.1.0',
+    MAJOR_VERSION: 7,
+    MIN_VERSION: 1,
     PATCH_VERSION: 0,
```
Comment (enahum) [inline app/constants/calls.ts:12]: I think, this does not have to be changed.
cc: @cpoile

Comment (cpoile) [reply app/constants/calls.ts:12]: Correct -- it won't matter, considering v2's required server version is 7.1.0. This code (and the checks) could even be removed. I'll file a ticket.

**Review by cpoile — APPROVED** [review]
@amyblais you can leave the calls-related code unchanged, and I'll remove it altogether in another PR. Thanks!

**Comment by enahum** [comment]
@cpoile don't remove it just yet as the 7.1 is not enforced but is an annoying reminder.

You can keep the 6.3 check for now

**Comment by cpoile** [comment]
@enahum Ah ok, thanks.

**Comment by amyblais** [comment]
@esethna Please help review by tomorrow's feature freeze deadline.

**Review by esethna — APPROVED** [review]

**Pull request merged**